active_record_shards.gemspec

```diff
@@ -10,6 +10,7 @@ Gem::Specification.new "active_record_shards", "3.19.1" do |s|
 
   s.add_runtime_dependency("activerecord", ">= 5.0", "< 6.1")
   s.add_runtime_dependency("activesupport", ">= 5.0", "< 6.1")
+  s.add_runtime_dependency("ruby2_keywords")
```

Comment:
Maybe worth leaving a TODO to remove when Ruby < 2.7 support is dropped? (FWICT Rails never added it as a dependency, but did have conditional checks that were removed once obsolete - ref. https://github.com/rails/rails/commit/97b87dcff0e57bc31e50b27e81641eb390a3b8e7, https://rubyreferences.github.io/rubychanges/2.7.html#keyword-argument-related-changes)

Reply:
Huh, I actually kinda like the Rails approach where they didn’t have to add a runtime dependency. I’ll change this PR.

Reply:
> Maybe worth leaving a TODO to remove when Ruby < 2.7 support is dropped?

🤷🏼 We’ll be reminded when newer Ruby versions remove the `.ruby2_keywords` method.